akka/shopping-cart-java/pom.xml

```diff
@@ -44,7 +44,7 @@
             <dependency>
                 <groupId>com.lightbend.akka</groupId>
                 <artifactId>akka-dependencies_${scala.binary.version}</artifactId>
-                <version>23.05.4</version>
+                <version>23.10.0</version>
```

Comment:
Need the `23.10.1` version of the akka-dependencies BOM published, that will include Cinnamon 2.19.0.